docs(github) Updates to doc contribution instructions (#2198)
Kong/docs.konghq.com: CONTRIBUTING.md

[Back to TOC](#table-of-contents)

### Formatting Documentation

> The Kong docs site has a number of features built in to better display content.

Reviewer: ```suggestion
The Kong docs site has a number of features built-in to better display content.
```

https://www.merriam-webster.com/dictionary/built-in

Author: It's not a noun or an adjective, but a phrasal verb in this situation. I can rephrase to avoid it, though.

#### Markdown Front Matter

Markdown files on the doc site (excluding `docs.konghq.com/hub/`) must have a
yaml front matter section with at least one parameter (`title`). You can also

> specify additional parameters to change how your doc will display.

Reviewer: "display" is a transitive verb that requires an object.  Either the fix is state the object, use passive, or say "renders" instead.

```suggestion
specify additional parameters to change how your doc is displayed.
```

On initial page load, the first tab ("<your title here>" in the example above)
will be the one displayed.

> > **Note:** You can’t nest tabs within tabs.

Reviewer: ```suggestion
> **Note:** You can’t nest navtabs within navtabs.
```

Suggesting using the command name because I am correcting this syntax in plugin docs. 

![image](https://user-images.githubusercontent.com/3756245/88809828-2b3f3780-d17a-11ea-99bf-61b6e9b5cdc0.png) from https://github.com/Kong/docs.konghq.com/commit/24378ad662de5780feca9a16708e688f734c8ce7#diff-e7bfaf0d7282472697dbb2f05400a498L163

Do you want such an indent and gray sidebar for the note?

![image](https://user-images.githubusercontent.com/3756245/88809562-e5826f00-d179-11ea-9c8e-27dd28fa66ce.png)

Author: Re: note formatting, that's the way GitHub renders notes, so I don't have any issues with it.
